corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -223,7 +229,8 @@ def remove_grant(self, priv_slug, dry_run=False):
     ]
 
     BOOTSTRAP_GRANTS = {
-        'community_plan_v0': community_plan_features,
+        'community_plan_v0': community_plan_v0_features,
+        'community_plan_v1': community_plan_v1_features,
```

Comment:
No such role currently exists, but added an assertion just in case: https://github.com/dimagi/commcare-hq/pull/12821/commits/7d7d5406fde5be3bc42aad3015d5eb26fdea5e57